README.md

```diff
@@ -16,3 +16,16 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+o states that a function grows slower than some order of magnitude
+
+Meaning that this is saying if g(n) is growing faster than f(n) if and only if f(n) < g(n)
+
+$f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$  that will always be true
```

Comment:
Not following you here -- surely whether this is true depends on $g(n)$ and $f(n)$?